pkg/monitoring/alloy/templates/alloy-config.alloy.template

```diff
@@ -19,6 +19,9 @@ mimir.rules.kubernetes "{{ . }}" {
         value = "{{ $.ClusterID }}"
     }
   }
+  {{- else }}
+  address = "http://mimir-ruler.mimir.svc:8080/"
+  {{- end }}
   mimir_namespace_prefix = "{{ $.ClusterID }}"
```

Comment:
Have you tested how it looks like when importing the exact same rule from 2 clusters using the same tenant ? Are  the 2 rules working independently as expected ?

Reply:
The expressions are edited to contain cluster_id=cluster_id in there. Also, they are namespaced under cluster_id_1/namespace and cluster_id_2/namespace so they are independent. The only issue we have is when we have the same rule on the WC and on MC because we don't have the notion of cluster scope on the MC